shared/triage_metadata_test.go

```diff
@@ -250,6 +252,94 @@ links:
 	assert.Equal(t, "a.html", actual.Links[2].Results[0].TestPath)
 }
 
+func TestAddToFiles_AddNewMetadataLink_Label(t *testing.T) {
+	var amendment MetadataResults
+	json.Unmarshal([]byte(`{
+		"/foo/foo1/abc.html": [
+			{
+				"label": "interop"
+			}
+		]
+	}`), &amendment)
+
+	var path = "foo/foo1"
+	var fileMap = make(map[string]Metadata)
+	fileInBytes := []byte(`
+links:
+  - product: chrome
+    url: foo
+    results:
+    - test: b.html
+  - product: firefox-2
```

Comment:
Is the trailing "-2" here significant? I guess it means Firefox version 2, and in the expectations below the product is just "firefox", but I guess this is already covered by other tests?

Reply:
I copied and pasted from old tests. Removed them all :)